mobile/src/home-view.js

```diff
@@ -176,6 +175,26 @@ export default class HomeView extends Component {
     )
   }
 
+  renderScanSuccess() {
```

Comment:
These 2 functions are not used.

Reply:
These functions are for future use to resolve a QAS ticket